reconcile/utils/terrascript_client.py

```diff
@@ -1527,7 +1527,7 @@ def populate_tf_resource_s3(self, resource, namespace_info):
             tf_resources.append(sqs_data)
 
             s3_events = common_values.get('s3_events', ["s3:ObjectCreated:*"])
-            notification_values = {
+            notification = {
```

Comment:
Eventually this part will be deleted, and its functionality will be take care by the loop added at line 1549. This is mostly trading duplicated code for backwards compatibility. Plan to migrate the date, then consolidate the code.